test/expected/size_utils.out

```diff
@@ -153,6 +153,30 @@ SELECT * FROM chunks_detailed_size('toast_test');
  _timescaledb_internal | _hyper_4_9_chunk |       24576 |       16384 |        8192 |       49152 | 
 (1 row)
 
+-- Ensure approximate_row_count() works both for regular table and hypertable
+--
+CREATE TABLE approx_count(time TIMESTAMP, value int);
+INSERT INTO approx_count VALUES('2004-01-01 10:00:01', 1);
+INSERT INTO approx_count VALUES('2004-01-01 10:00:02', 2);
+INSERT INTO approx_count VALUES('2004-01-01 10:00:03', 3);
+INSERT INTO approx_count VALUES('2004-01-01 10:00:04', 4);
+INSERT INTO approx_count VALUES('2004-01-01 10:00:05', 5);
+INSERT INTO approx_count VALUES('2004-01-01 10:00:06', 6);
+INSERT INTO approx_count VALUES('2004-01-01 10:00:07', 7);
+SELECT * FROM approximate_row_count('approx_count');
+ approximate_row_count 
+-----------------------
+                     0
+(1 row)
+
+ANALYZE approx_count;
+SELECT * FROM approximate_row_count('approx_count');
+ approximate_row_count 
+-----------------------
+                     7
+(1 row)
+
+DROP TABLE approx_count;
```

Comment:
I guess you need not drop the table here, you can just do create_hypertable with migrate_data=>true

Reply:
I did it because did not want to mess with saved statistics by previous ANALYZE call